### ISSUE TITLE
HDD-OSD PS1 install doesn't work with English display language

### ISSUE THREAD

**Issue by zulc22** [GDX-X/PFS-BatchKit-Manager#11]
```
File is too small to process
         VCD's should be at least 1104128 bytes in length
```
My VCD files are not too small or too large, and they work with the regular install method.

**Comment by GDX-X** [comment]
Hey, 

What is the game you want to install

Does your VCD exceed 1gb?

**Comment by zulc22** [comment]
I have about 20 games, none of them work, and none of the VCDs exceed 1GB.

**Comment by zulc22** [comment]
I don't know what logic is actually wrong with your version, but [I re-implemented the process](https://gist.github.com/zulc22/cc50af26581b44443934ec132b2d2218) myself in a way that didn't cause any errors for me.

**Comment by israpps** [comment]
Probably a `POPS-VCD-ID-EXTRACTOR.EXE` issue

**Comment by israpps** [comment]
Try processing your games with the latest build
https://github.com/israpps/POPS-VCD-ID-Extractor/releases/download/Latest/POPS-VCD-ID-Extractor.exe

I changed cout to print the file size that was obtained

**Comment by GDX-X** [comment]
> I don't know what logic is actually wrong with your version, but [I re-implemented the parts of the process that fail](https://gist.github.com/zulc22/cc50af26581b44443934ec132b2d2218) myself in a way that didn't cause any errors for me.

Yes it is normal because you do not use `POPS-VCD-ID-EXTRACTOR` which allows to obtain the identifier of the VCDs. which allows you to download the ARTs and have other things. when you install the VCDs in a __.POPS partition, I do not call `POPS-VCD-ID-EXTRACTOR` so this problem does not appear

**Comment by zulc22** [comment]
https://ia601507.us.archive.org/view_archive.php?archive=/28/items/OPLM_ART_2022_01/OPLM_ART_2022_01.zip&file=PS1%2FCPCS_007.01%2FCPCS_007.01_COV.jpg no longer exists.

**Comment by GDX-X** [comment]
Yes for the ART it will be corrected in the new version because the ip of archive.org changes every time.

Did you convert your PS1 games with my script or another program?

**Comment by zulc22** [comment]
I added some diagnostic code to my copy of PFS BatchKit, and the program appears to _not copy the .VCD file_ before attempting to read it, and the `!filename!` field is EMPTY!
```
>dir "C:\Users\zulc2\Downloads\PFS_BatchKit_Manager_v1.0.8\POPS\temp\96 Arthur! Ready to Race"
 Volume in drive C has no label.
 Volume Serial Number is FCFB-A4DE

 Directory of C:\Users\zulc2\Downloads\PFS_BatchKit_Manager_v1.0.8\POPS\temp\96 Arthur! Ready to Race

03/07/2022  07:38 AM    <DIR>          .
03/07/2022  07:37 AM    <DIR>          ..
               0 File(s)              0 bytes
               2 Dir(s)  353,987,526,656 bytes free

>"C:\Users\zulc2\Downloads\PFS_BatchKit_Manager_v1.0.8\BAT\POPS-VCD-ID-Extractor" "C:\Users\zulc2\Downloads\PFS_BatchKit_Manager_v1.0.8\POPS\temp\96 Arthur! Ready to Race\"

File is [0] bytes in length
         VCD's should be at least [1104128] bytes in length to be processed
```

**Comment by GDX-X** [comment]
Can you try it on another pc or a virtual machine I just tried I don't have this bug for any PS1 games

?

**Comment by zulc22** [comment]
```
cd /d "%~dp0POPS\temp\%%a" & for %%V in (*.VCD) do set filename=%%V & cd /d "%~dp0POPS\temp"
```
This is the line that fails. the for loop doesn't loop through anything.

**Comment by GDX-X** [comment]
Yes it's weird that on your machine he can't find it. maybe your anti-virus is interfering with busybox?

**Comment by GDX-X** [comment]
Mhh I think something on your machine must be interfering with it but I don't know what can cause this

what system do you have windows 10 64 bit?

**Comment by zulc22** [comment]
Windows 11.

**Comment by GDX-X** [comment]
This may be a Windows 11 specific issue. and busybox, I've never tried it on it

**Comment by zulc22** [comment]
> Yes for the ART it will be corrected in the new version because the ip of archive.org changes every time.

That's not an IP, it's a server. You have to use the REAL link that the pages reference. Archive just redirects you.
Use https://archive.org/download/OPLM_ART_2022_01/OPLM_ART_2022_01.zip/PS1%2FCPCS_007.01%2FCPCS_007.01_COV.jpg instead.

**Comment by zulc22** [comment]
It does not work in a Windows 10 VM either.
![image](https://user-images.githubusercontent.com/54846511/157090753-56f8adc0-4b59-4dff-9cff-eea178de09f5.png)

**Comment by GDX-X** [comment]
> > Yes for the ART it will be corrected in the new version because the ip of archive.org changes every time.
> 
> That's not an IP, it's a subdomain that only points to the correct server SOME of the time. You have to use the REAL link that the pages reference. Archive just redirects you. Use https://archive.org/download/OPLM_ART_2022_01/OPLM_ART_2022_01.zip/PS1%2FCPCS_007.01%2FCPCS_007.01_COV.jpg instead.

Thanks for that





> It does not work in a Windows 10 VM either. ![image](https://user-images.githubusercontent.com/54846511/157090753-56f8adc0-4b59-4dff-9cff-eea178de09f5.png)

mhhh So it does that to you for all your games? 

you may be missing microsoft visual c++ try to install this pack on your virtual machine
https://anonfiles.com/L0X6ndI1xc/Microsoft_Visual_C_Pack_v5.9_Repack_zip

**Comment by zulc22** [comment]
that is an iso image that has a shortcut that refers to a file i don't have.
![image](https://user-images.githubusercontent.com/54846511/157094332-e1d5b33e-1334-423f-99f3-c31cd6a8daf4.png)

**Comment by zulc22** [comment]
okay, DO NOT upload to anonfiles. it only gives me the real download if I have a fucking ad-blocker installed.

**Comment by zulc22** [comment]
also, are you sure that the version you have is identical to the github version and not a modified, unreleased copy? mine is

**Comment by GDX-X** [comment]
No I just downloaded to check that everything is fine and for me everything works. you are the first one this problem has happened to. at the moment I only see VCDs as a problem

Try to convert a BIN/CUE PS1 with my script to .VCD to be sure

**Comment by zulc22** [comment]
> No I just downloaded to check that everything is fine and for me everything works. you are the first one this problem has happened to. at the moment I only see VCDs as a problem
> 
> Try to convert a BIN/CUE PS1 with my script to .VCD to be sure

I already did, I converted my VCDs (converted by PSXVCD) to BIN/CUE inside PFSBK without issue and just to be sure converted them back to VCD within PFSBK instead of PSXVCD.

**Comment by GDX-X** [comment]
Yes try I will try

**Comment by zulc22** [comment]
```

File is [0] bytes in length
         VCD's should be at least [1104128] bytes in length to be processed


----------------------------------------------------
Completed...


Press any key to continue . . .
```
With only one VCD in the POPS directory, which was created by PFSBK.

`Bubble Bobble - featuring Rainbow Islands.VCD`
SHA256 C3B0D4D340577CF0A1A26C06F230B7176D9EBFE0C0173FC34478E6B701261107

**Comment by zulc22** [comment]
a workaround is to change your system language to french

**Comment by GDX-X** [comment]
Yes it's fixed will be out soon 1.0.9

**Comment by israpps** [comment]
> @israpps
> 
> > I changed cout to print the file size that was obtained
> 
> not really related to the issue at hand, but maybe you should also have the program check that the filename that was given actually refers to a file that exists (in this case, it doesn't)

if the file does not exist, size reported would be `-1`
